src/components/intersection.js

```diff
@@ -1,5 +1,40 @@
 import * as THREE from 'three';
 
+export const CROSSWALKS = {
+  none: 0,
+  'crosswalk-zebra': 1,
+  'crosswalk-raised': 2,
+  'crosswalk-rainbow': 3,
+  'crosswalk-double': 4,
+  'crosswalk-mural': 5,
+  'crosswalk-piano': 6
+};
+export const CROSSWALKS_REV = {};
+Object.keys(CROSSWALKS).forEach((key) => {
+  CROSSWALKS_REV[CROSSWALKS[key]] = key;
+});
+
+const transformImageCrosswalk = (entity, length, rotZ) => {
+  entity.setAttribute('rotation', { z: rotZ });
+  entity.setAttribute('scale', { y: length / 12, x: 1.5 });
```

Comment:
Please define all three x y z. Although that somehow works on aframe 1.5.0, setting a partial vec3 is not a valid usage and sometimes won't work properly (setting missing values to 0 and not 1 as you expect for scale) on aframe 1.6.0 or later depending on if you insert the entity in DOM first.